### DISCUSSION TITLE
How does Ungoogled Chromium works ?

### DISCUSSION THREAD

**Discussion by Rexadev** [Eloston/ungoogled-chromium#2487] [General]
1. Is it first chromium is patched and everyone builds their own way with - complier cpu optimizations, codecs, google sync API ?

2. So [macchrome/winchrome](https://github.com/macchrome/winchrome)'s and all other "Ungoogle Chromium" is downstream from [ungoogled-software/ungoogled-chromium](https://github.com/ungoogled-software/ungoogled-chromium).

3. Do [ungoogled-software/ungoogled-chromium](https://github.com/ungoogled-software/ungoogled-chromium) repo run https://chromium.woolyss.com ?

4. Is Bromite, Chromite downstream for [ungoogled-software/ungoogled-chromium](https://github.com/ungoogled-software/ungoogled-chromium) repo too ?

5. What other projects downstream from here ?

6. Where does https://github.com/Eloston fit in ungoogled-chromium

**Comment by tomasz1986** [comment]
> 3\. Do this repo run [chromium.woolyss.com](https://chromium.woolyss.com) ?

The question should rather be whether they use this repo as their source, but the answer is that they do not. The Ungoogled Chromium builds that they recommend mostly come from 3rd party sources and aren't supported here.

**Edit**:

Now I see what you mean. The answer is that that website is completely unrelated to this repo.

> 4\. Is Bromite, Chromite downstream for this repo too ?

These are separate projects unrelated to Ungoogled Chromium.
